scripts/kind-with-registry.sh

```diff
@@ -40,10 +40,38 @@ if [ "${running}" != 'true' ]; then
   docker run -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" registry:2
 fi
 
+# Set up the key pair file paths for azwi in CI
+# SERVICE_ACCOUNT_SIGNING_PUB and SERVICE_ACCOUNT_SIGNING_KEY are preset envs
+# Refer: https://github.com/kubernetes/test-infra/pull/28303/files
+
+pub=$(echo ${SERVICE_ACCOUNT_SIGNING_PUB} | base64 --decode)
```

Comment:
@CecileRobertMichon -- Getting this error here. 
```
./scripts/kind-with-registry.sh: line 47: SERVICE_ACCOUNT_SIGNING_PUB: unbound variable
```
How can I convert what is available in in the env to a file so that kind cluster gets to mount this file. Also is this env accessible for the process launched by this script?

Reply:
Oh.. Never mind. I got confused. So the kind(bootstrap cluster ) for e2e test are created via the capi library and looks like there is not option to pass the flags for kind cluster that is needed.